assets/js/components/IngestSheet/Completed.jsx

```diff
@@ -84,7 +61,7 @@ const IngestSheetCompleted = ({ sheetId, title }) => {
   return (
     <div>
       {ingestSheetErrors.length === 0 && (
-        <IngestSheetDetails totalWorks={totalWorks} />
+        <IngestSheetDetails totalWorks={workCountData.ingestSheetWorkCount} />
```

Comment:
Looks great.  Would there ever not be a `ingestSheetWorkCount`, or will it always return something (`null` or `0`)?

Reply:
Query returns an integer, so it would be zero. Is that an issue?

<img width="1339" alt="Screen Shot 2021-03-04 at 8 30 17 AM" src="https://user-images.githubusercontent.com/6372022/109987459-e0fbd700-7cc3-11eb-9d10-2e8504930f04.png">

Reply:
As long as it always returns some type of truthy value (even though it could be a 0 here), then yea should be all good.